examples/cra/jest.config.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import type { Config } from "@jest/types";
+import { pathsToModuleNameMapper } from "ts-jest";
+// Load the config which holds the path aliases.
+import { compilerOptions } from "./tsconfig.paths.json";
+
+const config: Config.InitialOptions = {
+  preset: "ts-jest",
+  testEnvironment: "jsdom",
+  moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, {
+    // This has to match the baseUrl defined in tsconfig.json.
+    prefix: "<rootDir>/../../",
```

Comment:
This needs to be just `"<rootDir>/"` to match the `baseUrl` in [`tsconfig.paths.json`](https://github.com/NiGhTTraX/ts-monorepo/blob/91cb6bf123aad0b6ddad38b4cf9da19794a1721d/examples/cra/tsconfig.paths.json#L3) (the CRA example contains both monorepo and package path aliases so it has its own config). Also, the comments in `tsconfig.paths.json` should be removed because the `import` doesn't like them.

Reply:
Excellent